mock-caas/src/main/java/com/sequenceiq/caas/grpc/service/MockUserManagementService.java

```diff
@@ -162,6 +171,21 @@ public GetEventGenerationIdsResponse load(String key) throws Exception {
                         return respBuilder.build();
                     }
                 });
+
+        initializeActorWorkloadCredentials();
+    }
+
+    @VisibleForTesting
+    void initializeActorWorkloadCredentials() {
+
+        GetActorWorkloadCredentialsResponse.Builder builder = GetActorWorkloadCredentialsResponse.newBuilder();
+        try {
+            JsonFormat.parser().merge(Resources.toString(
+                Resources.getResource("mock-responses/ums/get-workload-credentials.json"), StandardCharsets.UTF_8), builder);
```

Comment:
also looks like maybe you forgot to check this in?

Reply:
added.